### ISSUE TITLE
projectDocuments option not working

### ISSUE THREAD

**Issue by eu81273** [krisztianb/typedoc-plugin-merge-modules#17]
TypeDoc v0.26 added [projectDocuments](https://typedoc.org/options/input/#projectdocuments) option to support including external markdown pages.
https://typedoc.org/example/documents/External_Markdown.html

But with this plugin, projectDocuments option is not working at all.

**Comment by krisztianb** [comment]
Hi. This sounds like something I overlooked in the last major TypeDoc update. Do you have a repository with an example?

**Comment by dbohdiev** [comment]
Facing the same issue. Here is the config I'm using:

```
  "$schema": "https://typedoc.org/schema.json",

  "entryPoints": [
    "src/app/directives/",
    "src/app/services/",
    "src/app/utils/",
  ],
  "entryPointStrategy": "expand",
  "projectDocuments": ["documentation/*.md"],
  "out": "./docs",

  "tsconfig": "tsconfig.app.json",
  "plugin": ["typedoc-plugin-merge-modules"]
```

**Comment by krisztianb** [comment]
Hi guys. I released **version 6.0.1** which should no longer remove project documentations. 🎉
Please test the new version and report back if it is working as expected for your use cases. Thank you.

**Comment by krisztianb** [comment]
No complaints? I guess this is done then. 😄
